src/hb/tag_table.rs

```diff
@@ -1684,7 +1684,7 @@ fn lang_matches(language: &str, spec: &str) -> bool {
 fn strncmp(s1: &str, s2: &str, n: usize) -> bool {
     let n1 = core::cmp::min(n, s1.len());
     let n2 = core::cmp::min(n, s2.len());
-    s1[..n1] == s2[..n2]
+    &s1[..n1] == &s2[..n2]
```

Comment:
It's because I reran the Python script, and there it's with a reference: https://github.com/harfbuzz/rustybuzz/blob/8c52723ff75e91a33ae36e527baed871097e64bf/scripts/gen-tag-table.py#L983